src/main/webapp/js/common.js

```diff
@@ -570,22 +594,30 @@ function setStatusMessage(message, status) {
     if (message === '' || message === undefined || message === null) {
         return;
     }
+    var $statusMessageDivToUser = populateStatusMessageDiv(message, status);
+    $statusMessageDivToUser.show();
+    scrollToElement($statusMessageDivToUser[0], { offset: window.innerHeight / 2 * -1 });
+}
+
+/**
+ * Sets a status message and the message status to a given form.
+ * Default message type is info.
+ *
+ * @param message the text message to be shown to the user
+ * @param status type
+ * @param form form which should own the status
+ */
+function setStatusMessageToForm(message, status, form) {
+    if (message === '' || message === undefined || message === null) {
+        return;
+    }
+    // Copy the statusMessage and prepend to form
+    var $copyOfStatusMessagesToUser = populateStatusMessageDiv(message, status).clone().show();
+    $(DIV_STATUS_MESSAGE).remove();
+    $(form).prepend($copyOfStatusMessagesToUser);
+    scrollToElement($copyOfStatusMessagesToUser[0], { offset: window.innerHeight / 2 * -1,
+                                                      duration: 1000 });
```

Comment:
Right now after the scroll, the status message is in the middle of the page; it'll be nicer if the status message is somewhere near the top after the scroll so the user can still see most of the question